cpplint.py

```diff
@@ -3182,20 +3182,33 @@ def CheckEnd(self, filename, clean_lines, linenum, error):
                     )
 
 
-class _WrappedInfo(_BlockInfo):
+class _WrapInfo(_BlockInfo):
     """Stores information about parentheses, initializer lists, etc.
     Not exactly a block but we do need the same signature.
     Needed to avoid namespace indentation false positives,
     though parentheses tracking would slow us down a lot
     and is effectively already done by open_parentheses."""
 
-    pass
+    def __init__(self, linenum: int, index: int):
+        _BlockInfo.__init__(self, linenum, seen_open_brace=False)
+        # The index on lines[starting_linenum] where the wrap started
+        self.starting_index = index
 
 
-class _MemInitListInfo(_WrappedInfo):
+class _MemInitListInfo(_WrapInfo):
     """Stores information about member initializer lists."""
 
-    pass
+    def __init__(self, linenum: int, index: int):
+        _WrapInfo.__init__(self, linenum, index)
+        # For `: a(b)`, a is the identifier
+        self.expecting_identifier = True
+        # Whether we are expecting the value that follows the identifier
+        self.expecting_braces = False
+        # Whether we are currently within braces
+        self.in_braces = False
+        # Only used when within braces
+        self.is_curly = False
+        self.open_brace_count = 0
```

Comment:
```python
    if self.is_curly:
        assert self.in_braces, "self.is_curly=True requires that self.in_braces is also True"
```

Reply:
I could convert is_curly to a method property with a setter function that validates, but is that really necessary? I don't envision a case in the code where we would set is_curly to True without the guarantee that in_braces is True. (Also, I can't just add the code you proposed into the constructor, as is_curly would always be False by that point.)